xbmc/cores/VideoPlayer/Edl.h

```diff
@@ -43,8 +33,10 @@ class CEdl
   int RemoveCutTime(int iSeek) const;
   double RestoreCutTime(double dClock) const;
 
-  bool InCut(int iSeek, Cut *pCut = NULL);
-  bool GetNearestCut(bool bPlus, const int iSeek, Cut *pCut) const;
+  const std::vector<EDL::Cut>& GetCutList() const { return m_vecCuts; }
+
+  bool InCut(int iSeek, EDL::Cut *pCut = NULL);
```

Comment:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#104-null-vs-nullptr
```suggestion
  bool InCut(int iSeek, EDL::Cut* pCut = nullptr);
```